lib/white_bread/suite_configuration.ex

```diff
@@ -0,0 +1,68 @@
+defmodule WhiteBread.SuiteConfiguration do
+  alias WhiteBread.Suite.DuplicateSuiteError
+
+  defmacro __using__(_opts) do
+    quote do
+      import WhiteBread.SuiteConfiguration
+
+      @suites []
+      @before_compile WhiteBread.SuiteConfiguration
+    end
+  end
+
+  defmacro __before_compile__(_env) do
+    quote do
+      def suites do
+        @suites
+          |> throw_if_not_unique
```

Comment:
@rkotze the suites are reversed because the macro defining the suites pushes them on to the head of a list. So in order for them to be run in defining order (which I think is the least suprising thing) it needs to be reversed.

I've thought about this a bit more though and I may switch the macro to do a list append. It's slower  and not idiomatic but I think neither of these things is really in a downside in this context.